### ISSUE TITLE
Function _load_textdomain_just_in_time was called incorrectly

### ISSUE THREAD

**Issue by campbell-m** [trustedlogin/Remove-Dashboard-Access#41]
I’m getting the error message below when loading WordPress 6.7 using RDA Version 1.2 using

~~~php
define('WP_USE_THEMES', false);
require_once 'path/to/wordpress/wp-load.php';
~~~

~~~text
E_USER_NOTICE in /home/mydomain/public_html/wp-includes/functions.php at line 6114
Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>remove_dashboard_access</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/>Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.)
#0 MRBS\generate_backtrace() called at [/home/mydomain/public_html/book/functions_error.inc:228]
#1 MRBS\output_error() called at [/home/mydomain/public_html/book/functions_error.inc:274]
#2 MRBS\error_handler()
#3 trigger_error() called at [/home/mydomain/public_html/wp-includes/functions.php:6114]
#4 wp_trigger_error() called at [/home/mydomain/public_html/wp-includes/functions.php:6054]
#5 _doing_it_wrong() called at [/home/mydomain/public_html/wp-includes/l10n.php:1355]
#6 _load_textdomain_just_in_time() called at [/home/mydomain/public_html/wp-includes/l10n.php:1385]
#7 get_translations_for_domain() called at [/home/mydomain/public_html/wp-includes/l10n.php:194]
#8 translate() called at [/home/mydomain/public_html/wp-includes/l10n.php:339]
#9 esc_html__() called at [/home/mydomain/public_html/wp-content/plugins/remove-dashboard-access-for-non-admins/inc/class.rda-options.php:62]
#10 RDA_Options->setup() called at [/home/mydomain/public_html/wp-content/plugins/remove-dashboard-access-for-non-admins/inc/class.rda-options.php:43]
#11 RDA_Options->__construct() called at [/home/mydomain/public_html/wp-content/plugins/remove-dashboard-access-for-non-admins/remove-dashboard-access.php:27]
#12 include_once(/home/mydomain/public_html/wp-content/plugins/remove-dashboard-access-for-non-admins/remove-dashboard-access.php) called at [/home/mydomain/public_html/wp-settings.php:526]
#13 require_once(/home/mydomain/public_html/wp-settings.php) called at [/home/mydomain/public_html/wp-config.php:66]
#14 require_once(/home/mydomain/public_html/wp-config.php) called at [/home/mydomain/public_html/wp-load.php:50]
#15 require_once(/home/mydomain/public_html/wp-load.php) called at [/home/mydomain/public_html/book/auth/cms/wordpress.inc:9]
#16 require_once(/home/mydomain/public_html/book/auth/cms/wordpress.inc) called at [/home/mydomain/public_html/book/lib/MRBS/Session/SessionWordpress.php:8]
#17 require(/home/mydomain/public_html/book/lib/MRBS/Session/SessionWordpress.php) called at [/home/mydomain/public_html/book/lib/autoload.inc:21]
#18 {closure}()
#19 spl_autoload_call() called at [/home/mydomain/public_html/book/lib/MRBS/Session/SessionFactory.php:15]
#20 MRBS\Session\SessionFactory::create() called at [/home/mydomain/public_html/book/mrbs_auth.inc:35]
#21 MRBS\session() called at [/home/mydomain/public_html/book/init.inc:41]
#22 require_once(/home/mydomain/public_html/book/init.inc) called at [/home/mydomain/public_html/book/defaultincludes.inc:35]
#23 require(/home/mydomain/public_html/book/defaultincludes.inc) called at [/home/mydomain/public_html/book/index.php:11]
~~~

**Comment by zackkatz** [comment]
Thanks for reporting this, @campbell-m! Fixed in 1.2.1.

**Comment by campbell-m** [comment]
Thanks.  Yes, I can confirm it's fixed.
